intelmq/bots/experts/modify/examples/morefeeds.conf

```diff
@@ -3,9 +3,7 @@
     "__default": [{
             "classification.identifier": "",
             "malware.name": ".*"
-        }, {
-            "classification.identifier": "{msg[malware.name]}"
```

Comment:
See the rationale in https://github.com/certtools/intelmq/pull/663/commits/8a775c55041b78c73eedaa6f6bc112774226549c  It never worked reliably.
If this functionality is still needed, it can be covered by adding a second modify bot.